images-list

```diff
@@ -53,13 +53,23 @@ kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 1.1.0
 library/busybox rancher/library-busybox 1.31.1
 library/nginx rancher/library-nginx 1.19.2-alpine
 longhornio/csi-attacher rancher/longhornio-csi-attacher v2.0.0
+longhornio/csi-attacher rancher/longhornio-csi-attacher v2.2.1-lh1
 longhornio/csi-node-driver-registrar rancher/longhornio-csi-node-driver-registrar v1.2.0
+longhornio/csi-node-driver-registrar rancher/longhornio-csi-node-driver-registrar v1.2.0-lh1
 longhornio/csi-provisioner rancher/longhornio-csi-provisioner v1.4.0
+longhornio/csi-provisioner rancher/longhornio-csi-provisioner v1.6.0-lh1
 longhornio/csi-resizer rancher/longhornio-csi-resizer v0.3.0
+longhornio/csi-resizer rancher/longhornio-csi-resizer v0.5.1-lh1
+longhornio/csi-snapshotter rancher/longhornio-csi-snapshotter v2.1.1-lh1
```

Comment:
This repo doesn't exist, needs to be created first: https://hub.docker.com/r/rancher/longhornio-csi-snapshotter

Reply:
@cbron This is in the rancher org. Can someone in the Rancher team help to create those?